Doc/library/stdtypes.rst

```diff
@@ -5005,9 +5048,17 @@ All parameterized generics implement special read-only attributes.
 
 .. seealso::
 
-   * :pep:`585` -- "Type Hinting Generics In Standard Collections"
-   * :meth:`__class_getitem__` -- Used to implement parameterized generics.
-   * :ref:`generics` -- Generics in the :mod:`typing` module.
+   :pep:`484` - Type Hints
+      Introducing Python's framework for type annotations.
+
+   :pep:`585` - "Type Hinting Generics In Standard Collections"
```

Comment:
Dang, I just realized the quotation marks here are inconsistent with the rest of the docs (they should be removed).